corehq/apps/users/static/users/js/enterprise_users.js

```diff
@@ -31,20 +32,49 @@ hqDefine("users/js/enterprise_users", [
         // Only relevant for web users
         self.expanded = ko.observable(false);
 
+
         return self;
     };
 
     var enterpriseUsersList = function (options) {
+
         var self = webUsersList(options);
+
         self.toggleLoginAsUsers = function (webUser) {
             webUser.expanded(!webUser.expanded());
             _.each(self.users(), function (user) {
                 if (user.loginAsUser === webUser.username) {
-                    user.visible(webUser.expanded());
+                    if (!self.showDeactivated()) {
+                        user.visible(webUser.expanded() && user.is_active);
+                    }
+                    if (self.showDeactivated()) {
+                        user.visible(webUser.expanded() && !user.is_active);
+                    }
                 }
             });
         };
 
+        self.showDeactivated = ko.observable(false);
+
+        self.showHideDeactivated = function () {
```

Comment:
I keep stumbling over this name. Would it be accurate to call it `toggleDeactivated`?

Reply:
Yes, I think `toggleDeactivated` would be accurate and more clear. `showHideDeactivated` was kind of a working title I was using, and I can see how it's awkward to read.